devfiles/go/devfile.yaml

```diff
@@ -25,51 +36,79 @@ components:
 -
   type: dockerimage
   # this version is used in the plugin
-  image: quay.io/eclipse/che-golang-1.12:nightly
+  image: quay.io/eclipse/che-golang-1.14:nightly
   alias: go-cli
   env:
     - name: GOPATH
       # replicate the GOPATH from the plugin
-      value: /go:$(CHE_PROJECTS_ROOT)
+      value: $(CHE_PROJECTS_ROOT)
     - name: GOCACHE
       # replicate the GOCACHE from the plugin, even though the cache is not shared
       # between the two
       value: /tmp/.cache
   endpoints:
     - name: '8080-tcp'
       port: 8080
-  memoryLimit: 512Mi
+  memoryLimit: 2Gi
   mountSources: true
 commands:
 -
-  name: run outyet
+  name: '1.1 Run outyet'
   actions:
   - type: exec
     component: go-cli
     command: go get -d && go run main.go
     workdir: ${CHE_PROJECTS_ROOT}/src/github.com/golang/example/outyet
 -
-  name: stop outyet
+  name: '1.2 Stop outyet'
   actions:
   - type: exec
     component: go-cli
     command: kill $(pidof go)
 -
-  name: test outyet
+  name: '1.3 Test outyet'
   actions:
   - type: exec
     component: go-cli
     command: go test
     workdir: ${CHE_PROJECTS_ROOT}/src/github.com/golang/example/outyet
 -
-  name: run current file
+  name: '2.1 xenisa :: install dependencies'
+  actions:
+  - type: exec
+    component: go-cli
+    command: go mod download
+    workdir: ${GOPATH}/src/github.com/xesina/golang-echo-realworld-example-app
+-
+  name: '2.2 xenisa :: run'
+  actions:
+  - type: exec
+    component: go-cli
+    command: go run main.go
+    workdir: ${GOPATH}/src/github.com/xesina/golang-echo-realworld-example-app
+-
+  name: '2.3 xenisa :: build'
+  actions:
+  - type: exec
+    component: go-cli
+    command: go build
+    workdir: ${GOPATH}/src/github.com/xesina/golang-echo-realworld-example-app
+-
+  name: '2.4 xenisa :: test'
+  actions:
+  - type: exec
+    component: go-cli
+    command: go test ./...
+    workdir: ${GOPATH}/src/github.com/xesina/golang-echo-realworld-example-app
+-
+  name: 'Run current file'
   actions:
   - type: exec
     component: go-cli
     command: go get -d && go run ${file}
     workdir: ${fileDirname}
 -
-  name: Debug current file
+  name: 'Debug current file'
```

Comment:
Why the additional `'` marks?

Reply:
It's a cosmetic decision. Just to be consistent with names of other command.